ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.view.js

```diff
@@ -147,6 +153,15 @@ module.exports = Marionette.LayoutView.extend({
     this.determineInput()
   },
   transformValue: function(value, comparator) {
+    const comparatorIndex = numberComparators.indexOf(
+      this.model.get('comparator')
+    )
+    if (comparatorIndex > 0) {
```

Comment:
what is happening here?

Reply:
So for non-numerical searches (primary example would be string attributes) the default value is set at '*' for the textbox when the value[0] is null, however, if a numerical search is done then we want to show '0' as the default value for the textbox instead of '*'.
It ensures that it looks like this for specific types of attributes (string vs numerical)
<img width="643" alt="screen shot 2019-02-05 at 3 15 54 pm" src="https://user-images.githubusercontent.com/23177404/52307839-f4a7a880-2958-11e9-8e86-d88ad2d6d9e8.png">
<img width="596" alt="screen shot 2019-02-05 at 3 15 45 pm" src="https://user-images.githubusercontent.com/23177404/52307842-f4a7a880-2958-11e9-86f6-12c0e7408b15.png">